oops/backplane/ring.py

```diff
@@ -654,21 +654,24 @@ def ring_radial_resolution(self, event_key):
     return self.register_backplane(key, resolution)
 
 #===============================================================================
-def ring_angular_resolution(self, event_key):
+def ring_angular_resolution(self, event_key, units="rad"):
     """Projected angular resolution in radians/pixel at the ring intercept.
 
     Input:
         event_key       key defining the ring surface event. Alternatively, a
                         ring_radius or radial_mode backplane key, in which case
                         this backplane inherits the mask of the given backplane
                         array.
+        units           longitude representation; "rad" or "km"
     """
 
+    assert units.lower() in ["rad", "km"]
```

Comment:
"assert" doesn't accomplish what we want. Please do the check and raise ValueError. ValueError is defined to be the error raised by any Python function in which the user passes an invalid or incorrect value. From https://docs.python.org/3/library/exceptions.html:

exception ValueError
Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.